drivers/i2c/i2c_mspm0g3xxx.c

```diff
@@ -636,6 +636,8 @@ static int i2c_mspm0g3xxx_init(const struct device *dev)
 	DL_I2C_enablePower((I2C_Regs *)config->base);
 	delay_cycles(POWER_STARTUP_DELAY);
 
+	DL_I2C_disableTargetWakeup((I2C_Regs *)config->base);
```

Comment:
This is only relevant for target mode, so it should be set with the other target configs. I'm aware it's not the case on the TI branch, but our init method is already different due to the "target-mode-only" config.
It should not be racy as long as set before enabling the target mode :)